### PR TITLE
Update: color and line-height removed

### PR DESCRIPTION
**Pull request by ejaz-tapestry**
Ticket: https://jira.tapestry.support/browse/TORO-54724 <br>

Extra line-height and color classes are removed

### CODE REVIEW

**Review by p-musi — CHANGES_REQUESTED** [review]
Hi @ejaz-tapestry

Please review comments.

For the black/white/grey colors, I'm looking to have 6 token values for backwards compatibility.

"color-black-base"
"color-black"
"color-white-base"
"color-white"
"color-grey-base"
"color-grey"

File: src/global/color/index.json [hunk 1]
```diff
@@ -79,97 +79,19 @@
...
     },
     "grey": {
       "base": {
         "value": "#808080"
       },
-      "10": {
-        "value": "#E5E5E5"
-      },
       "20": {
         "value": "#CCCCCC"
```
[the remark this answers is not in the document]
Comment (ejaz-tapestry) [reply src/global/color/index.json:89]: This value is in use I got an error in the build process so I left this as it is.

File: scripts/figma-dt/getBrandJson.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ const getFigmaPluginJson = (brand) => {
                 /font-[0-9]+/,
                 /font-track/,
                 /linear-gradient/,
+                /color-carousel-indicator-bg/,
```
[the remark this answers is not in the document]
Comment (ejaz-tapestry) [reply scripts/figma-dt/getBrandJson.js:55]: it was removed in the Black listing, SO I did not change it as it was matched and taken from the old motif File, But If you say we can add these back. and the last one is `color-carousel-indicator-bg` removed on request in this current ticket.

File: src/global/color/index.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (p-musi) [reply src/global/color/index.json:89]: Can we replace the use of grey.20.value with grey.base.value in the toggle.json file that uses it?  Then we can remove 20 here.

**Pull request closed without merging**